src/groups/mqb/mqba/mqba_clientsession.cpp

```diff
@@ -2979,27 +3408,14 @@ void ClientSession::invalidate()
     BSLS_ASSERT_SAFE(!dispatcher()->inDispatcherThread(this));
 
     dispatcher()->execute(
-        bdlf::BindUtil::bind(&ClientSession::invalidateDispatched, this),
+        bdlf::BindUtil::bindS(d_state.d_allocator_p,
+                              &ClientSession::invalidateDispatched,
+                              this),
```

Comment:
This is another place where we don't use `ClientSessionDestructGuard`. We cannot acquire a shared resource to prevent destruction and invalidate it at the same time.